plugins/rapid7_intsights/help.md

```diff
@@ -833,142 +445,206 @@ Example output:
-|score|float|False|Score|10|
-|severity|string|False|Severity|Low|
-|sources|[]source|False|Sources|[ { "confidenceLevel": 2, "name": "Cyber Threat Alliance" } ]|
-|system_tags|[]string|False|System tags|["MyTag_1"]|
-|tags|[]string|False|Tags|["MyTag_1"]|
-|type|string|False|Type|Domains|
-|value|string|False|Indicator value|example.com|
-|whitelist|boolean|False|Whitelist|True|
-|status|string|False|Status|Active|
-|reported_feeds|[]reported_feed|False|Reported Feeds|[ { "ID": "SampleID", "Name": "AlienVault OTX", "ConfidenceLevel": 3 } ]|
-
+| :--- | :--- | :--- | :--- | :--- |
+|content|[]content|False|Response content|{ "content": [ { "cveId": "CVE-2020-7064", "cpe": [ { "Value": "cpe:2.3:a:php:php:*:*:*:*:*:*:*:*", "Range": { "VersionStartIncluding": "7.2.0", "VersionEndIncluding": "", "VersionStartExcluding": "", "VersionEndExcluding": "7.2.9" }, "Title": "Php", "VendorProduct": "Php Php" } ], "publishedDate": "2020-04-01T04:15:00.000Z", "updateDate": "2021-10-25T10:14:52.978Z", "severity": "Low", "intsightsScore": 36, "cvssScore": 5.4, "mentionsAmount": 39, "mentionsPerSource": { "PasteSite": 0, "HackingForum": 0, "InstantMessage": 0, "DarkWeb": 0, "ClearWebCyberBlogs": 0, "CodeRepositories": 9, "Exploit": 0, "SocialMedia": 30 }, "firstMentionDate": "2020-03-19T15:09:00.000Z", "lastMentionDate": "2021-07-22T20:41:00.000Z", "exploitAvailability": false, "vulnerabilityOrigin": [ "API" ], "relatedThreatActors": [], "relatedMalware": [], "relatedCampaigns": [] } ], }|
+  
 Example output:
 
 ```
 {
-  "first_seen": "2020-01-01T20:01:27.344Z",
-  "last_seen": "2020-01-01T20:01:27.344Z",
-  "last_update": "2020-01-01T20:01:27.344Z",
-  "related_campaigns": [],
-  "related_malware": [],
-  "related_threat_actors": [],
-  "score": 10,
-  "severity": "Low",
-  "sources": [
-    {
-      "ConfidenceLevel": 2,
-      "Name": "Cyber Threat Alliance"
-    }
-  ],
-  "system_tags": [],
-  "tags": [],
-  "type": "Domains",
-  "value": "example.com",
-  "whitelist": true
+  "content": "{ \"content\": [ { \"cveId\": \"CVE-2020-7064\", \"cpe\": [ { \"Value\": \"cpe:2.3:a:php:php:*:*:*:*:*:*:*:*\", \"Range\": { \"VersionStartIncluding\": \"7.2.0\", \"VersionEndIncluding\": \"\", \"VersionStartExcluding\": \"\", \"VersionEndExcluding\": \"7.2.9\" }, \"Title\": \"Php\", \"VendorProduct\": \"Php Php\" } ], \"publishedDate\": \"2020-04-01T04:15:00.000Z\", \"updateDate\": \"2021-10-25T10:14:52.978Z\", \"severity\": \"Low\", \"intsightsScore\": 36, \"cvssScore\": 5.4, \"mentionsAmount\": 39, \"mentionsPerSource\": { \"PasteSite\": 0, \"HackingForum\": 0, \"InstantMessage\": 0, \"DarkWeb\": 0, \"ClearWebCyberBlogs\": 0, \"CodeRepositories\": 9, \"Exploit\": 0, \"SocialMedia\": 30 }, \"firstMentionDate\": \"2020-03-19T15:09:00.000Z\", \"lastMentionDate\": \"2021-07-22T20:41:00.000Z\", \"exploitAvailability\": false, \"vulnerabilityOrigin\": [ \"API\" ], \"relatedThreatActors\": [], \"relatedMalware\": [], \"relatedCampaigns\": [] } ], }"
```

Comment:
Could you please pretty print it?

Reply:
This was due to the example in the plugin spec being a string and not a dict, this has been updated